napari_allencell_annotator/util/image_utils.py

```diff
@@ -35,8 +35,8 @@ def __init__(self, filepath: Path):
         else:
             self._image = BioImage(filepath, reader=bioio_imageio.Reader)
 
-    def get_dask_data(self) -> da.Array:
+    def get_image(self) -> BioImage:
         """
-        Returns image data as a dask array
+        Returns Bioimage object loaded with correct reader
         """
-        return self._image.get_dask_stack()
+        return self._image
```

Comment:
Why was this changed back to an image?

Reply:
Changed back to dask stack in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/5a1eac3472eb10cd133a20dd0b9a299845829cba